### PR TITLE
 apps/btc: split BTCPub into xpub and address 

### PR DESCRIPTION
**Pull request by benma**

### CODE REVIEW

**Comment by benma** [comment]
The change starts in btc.proto, the rest is mostly mechanical.

File: src/commander/commander_btc.c [hunk 1]
```diff
@@ -28,70 +28,86 @@ static const char* _coin_tbtc = "BTC Testnet";
 static const char* _coin_ltc = "Litecoin";
 static const char* _coin_tltc = "LTC Testnet";
 
-commander_error_t commander_btc_pub(const BTCPubRequest* request, PubResponse* response)
+// Returns the string to be used in the confirm title. Returns NULL for an invalid coin.
+static const char* _coin_title(BTCCoin coin)
 {
-    if (!app_btc_enabled(request->coin)) {
```
Comment (TheCharlatan) [inline src/commander/commander_btc.c]: Why is this removed?

File: test/unit-test/test_app_btc.c [hunk 2]
```diff
@@ -95,167 +104,178 @@ bool __wrap_keystore_get_xpub(const uint32_t* keypath, size_t keypath_len, struc
...
+static void _test_app_btc_xpub(void** state)
+{
+    { // invalid coin
+        bool result = app_btc_xpub(_BTCCoin_MIN - 1, BTCPubRequest_XPubType_XPUB, NULL, 0, NULL, 0);
+        assert_false(result);
+        result = app_btc_xpub(_BTCCoin_MAX + 1, BTCPubRequest_XPubType_XPUB, NULL, 0, NULL, 0);
+        assert_false(result);
+    }
+
+    for (int bools = 0; bools < 8; bools++) {
```
Comment (TheCharlatan) [inline test/unit-test/test_app_btc.c]: wtf :D

File: src/commander/commander_btc.c (hunk 1 above)
Comment (benma) [reply src/commander/commander_btc.c]: The diff is just confusing, the line is still there in commander_btc_pub()

File: test/unit-test/test_app_btc.c (hunk 2 above)
Comment (benma) [reply test/unit-test/test_app_btc.c]: :innocent:
